__tests__/app.test.js

```diff
@@ -45,12 +45,47 @@ describe("GET /api", () => {
         .expect(200)
         .then(({ body }) => {
           const { endpoints } = body;
-          console.log(endpoints)
           expect(endpoints).toBeInstanceOf(Object);
           expect(endpoints).hasOwnProperty('description')
           expect(endpoints).hasOwnProperty('queries')
           expect(endpoints).hasOwnProperty('exampleResponse')
           expect(endpoints).toEqual(endpointsFile)
         });
     });
+})
+describe("GET /api/articles/:article_id", () => {
+    test("status:200, responds with a single article", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body;
+            console.log(article)
+            expect(article.article_id).toBe(2)
+            expect(article.title).toBe('Sony Vaio; or, The Laptop')
+            expect(article.topic).toBe('mitch')
+            expect(article.author).toBe('icellusedkars')
+            expect(article.body).toBe('Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.')
+            const date = new Date((1602828180000)-3600000)
```

Comment:
This is very thorough! you could have gotten away with expecting it to have a date property and just asserting that the id is correct and all properties are present